flang/lib/Lower/OpenMP/Utils.cpp

```diff
@@ -13,19 +13,22 @@
 #include "Utils.h"
 
 #include "Clauses.h"
+#include <DirectivesCommon.h>
+
+#include <flang/Evaluate/fold.h>
 #include <flang/Lower/AbstractConverter.h>
+#include <flang/Lower/ConvertExprToHLFIR.h>
```

Comment:
Are all of the header file changes necessary? Just making sure we only have the minimal set here.